test/Spago/Build/Polyrepo.purs

```diff
@@ -68,97 +71,189 @@ spec = Spec.describe "polyrepo" do
       <> imports
       <> body
 
-    mkMainModuleContent { modName, imports, logStatement, packageName } =
-      mkModuleContent
-        { modName
-        , imports:
-            [ ""
-            , "import Effect (Effect)"
-            , "import Effect.Console (log)"
-            ]
-              <> imports
-        , body:
-            [ ""
-            , "main :: Effect Unit"
-            , "main = do"
-            , "  " <> logStatement
-            ]
-              <>
-                ( packageName # maybe [] \pkgName ->
-                    [ ""
-                    , "packageName :: String"
-                    , "packageName = \"" <> pkgName <> "\""
-                    ]
-                )
-        }
+    mkDependencies :: Array String -> Config.Dependencies
+    mkDependencies = Config.Dependencies <<< Map.fromFoldable <<< map (flip Tuple Nothing <<< mkPackageName)
```

Comment:
Maybe this should go in `Test/Prelude.purs`?

Reply:
Absolutely!